[BUG] Missing Fetch message definition causes Makefile proto generation to fail

### Bug Description

While running the `make proto` command to generate C++ protobuf files for the Spark Connect C++ client, the build fails due to a missing protobuf message named `Fetch`. The file `spark/connect/ml.proto` references `"Fetch"` on line 34, but this message is not defined anywhere in the repository or included dependencies.

Expected: `make proto` should successfully generate all `.pb.cc` and `.pb.h` files.
Actual: The build stops with a protobuf compilation error indicating that `Fetch` is undefined.

### Steps to Reproduce

1. Clone the repository.
2. Run `make proto` inside the project root.
3. Observe the protobuf compilation failure.


### Expected Behavior

The protobuf generation step (`make proto`) should run to completion without undefined message references. All `.proto` dependencies should compile successfully.

### Actual Behavior

The process fails with:

```
spark/connect/ml.proto:34:5: "Fetch" is not defined.
make: *** [Makefile:129: build/gen/spark/connect/ml.pb.cc] Error 1
```

Indicating that the `Fetch` message referenced in `ml.proto` is missing.

### Minimal Reproducible Example

```cpp
git clone https://github.com/irfanghat/spark-connect-cpp
cd spark-connect-cpp
make proto

This triggers the `Fetch` not defined error.
```

### Environment

Ubuntu 22.04, protoc 3.x, GCC 11 / Clang 14, Spark Connect C++ Client (latest commit)

### Logs / Screenshots

```shell
user@dev:~/spark-connect-cpp$ make proto
protoc -I=src --cpp_out=build/gen src/spark/connect/base.proto
protoc -I=src --cpp_out=build/gen src/spark/connect/catalog.proto
protoc -I=src --cpp_out=build/gen src/spark/connect/commands.proto
protoc -I=src --cpp_out=build/gen src/spark/connect/common.proto
protoc -I=src --cpp_out=build/gen src/spark/connect/expressions.proto
protoc -I=src --cpp_out=build/gen src/spark/connect/ml.proto
spark/connect/ml.proto:34:5: "Fetch" is not defined.
make: *** [Makefile:129: build/gen/spark/connect/ml.pb.cc] Error 1
```

### Additional Context

The issue appears to be caused by a missing protobuf message definition. Other Spark Connect `.proto` files do not define `Fetch`, nor is it imported by `ml.proto`. This suggests `Fetch` was either removed, renamed, or intended to come from an upstream proto not included in the current repository.

--- a/src/spark/connect/ml.proto
+++ b/src/spark/connect/ml.proto
@@ -50,6 +50,20 @@ message MlCommand {
     Relation dataset = 3;
   }
 
+  // Command to fetch attributes or parameters from ML operators
+  // This can be used to retrieve specific attributes from estimators, evaluators, or cached models
+  message Fetch {
+    // ML operator information - can be an estimator/evaluator or a cached model
+    oneof type {
+      // Estimator or evaluator
+      MlOperator operator = 1;
+      // The cached model
+      ObjectRef obj_ref = 2;
+    }
+    // (Required) The name of the attribute to fetch
+    string attribute = 3;
+  }
+
   // Command to delete the cached objects which could be a model
   // or summary evaluated by a model
   message Delete {

